package.json

```diff
@@ -52,5 +52,9 @@
       "**/babel-loader",
       "**/webpack"
     ]
+  },
+  "dependencies": {
+    "file-saver": "^2.0.5",
+    "jszip": "^3.10.1"
```

Comment:
Should these have been added to devDependencies instead?

Reply:
I don't think it matters as much for `@penrose/editor`, but I think they should be in `dependencies`, as they are included in the final bundle in production.